src/utils/filter/getFilter.ts

```diff
@@ -45,15 +49,30 @@ const buildMongoFilter = (filter: any, fields: any[]): any => {
         let startDate: Date;
         let endDate: Date;
         let dateForFilter: any;
+        let startDatetime: Date;
+        let endDatetime: Date;
+        let datetimeForFilter: any;
```

Comment:
@Matthis-M-ReliefApps 
this is certainly a mistake, the datetimeforfilter never gets a value
and it is not visible in the other similar method getFilter